src/media/js/site/login.js

```diff
@@ -14,8 +14,39 @@ export class DefaultUnauthorizedHandler extends React.Component {
 };
 
 
+export class DefaultTOSSignatureHandler extends React.Component {
```

Comment:
I thought about that, but that would make the interaction between login, permissions, and TOS really complex. This has a nice, obvious flow:

https://github.com/mozilla/marketplace-content-tools/pull/28/files#diff-e498cbb0c349af1450e44801c52a047cL31